distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex.enso

```diff
@@ -428,6 +429,33 @@ type Regex
             if should_recompile.not then self else
                 Regex.compile self.pattern case_insensitive
 
+    ## Create a Regex from a glob string
+
+       Arguments:
+       - expression: The glob string to compile into a regex.
+    from_glob : Text -> Regex
+    from_glob expression:Text -> Regex =
+        to_escape = Hashset.from_vector ["/", "$", "^", "+", ".", "(", ")", "=", "|", "\"]
```

Comment:
This list seems insufficient.

E.g. `[` is lacking, but I guess also `{`, `?`.

How can we make sure the list is exhaustive?

Reply:
`[` is both glob and regex so pass throug
`{` is mapped to `(`
`?` is mapped to `.`

Reply:
Ah right

Reply:
Hmm but I guess we should be careful about `[]` - they handle negation differently (Regex: `[^a]` vs Glob: `[!a]`).

Reply:
Indeed and special case handled for `!`

Reply:
Ooops, sorry, I'm context switching between too many things and did not notice.

Looks ok then